git2consul/wp-config.json

```diff
@@ -0,0 +1,20 @@
+{
+	"config": [
+		{ "site_url": "http://my-site.example.com" },
+
+		{ "SAVEQUERIES":      true },
+		{ "WP_DEBUG":         true },
+		{ "WP_DEBUG_DISPLAY": true },
+
+		{ "AUTH_KEY":         "put your unique phrase here" },
+		{ "SECURE_AUTH_KEY":  "put your unique phrase here" },
+		{ "LOGGED_IN_KEY":    "put your unique phrase here" },
+		{ "NONCE_KEY":        "put your unique phrase here" },
+		{ "AUTH_SALT":        "put your unique phrase here" },
+		{ "SECURE_AUTH_SALT": "put your unique phrase here" },
+		{ "LOGGED_IN_SALT":   "put your unique phrase here" },
+		{ "NONCE_SALT":       "put your unique phrase here" },
+
```

Comment:
Perhaps punt on SMTP connection details, since the plugin I'm familiar with expects cornfugraiton via the dashboard: https://wordpress.org/plugins/wp-smtp/screenshots/